fuel-txpool/src/service.rs

```diff
@@ -10,9 +10,11 @@ pub struct ServiceBuilder {
     sender: txpool::Sender,
     receiver: mpsc::Receiver<TxPoolMpsc>,
     config: Config,
-    broadcast: broadcast::Sender<TxStatusBroadcast>,
+    consumer: broadcast::Sender<TxStatusBroadcast>,
     db: Option<Box<dyn TxPoolDb>>,
     import_block_events: Option<broadcast::Receiver<ImportBlockBroadcast>>,
+    broadcast_txs: Option<mpsc::Sender<TransactionBroadcast>>,
```

Comment:
Based on our earlier call today, I think we should use `P2pRequestEvent` instead of the TransactionBroadcast enum for sending. 

Another minor nit is that `broadcast_txs` isn't totally clear which direction the broadcast is going. Maybe something like `network_outbox` would help clarify the intention?

```suggestion
    network_outbox: Option<mpsc::Sender<P2pRequestEvent>>,
```

Reply:
I moved this to P2pRequestEvent, I do remember talking about this, I think it got mixed up between `fuel-core-interfaces/p2p/P2pRequestEvent::BroadcastNewTransaction` and `fuel-core-interfaces/p2p/TransactionBroadcast::NewTransaction` as both have similar names